src/cartservice/cartstore/LocalCartStore.cs

```diff
@@ -65,7 +65,7 @@ public Task AddItemAsync(string userId, string productId, int quantity)
         public Task EmptyCartAsync(string userId)
         {
             Console.WriteLine($"EmptyCartAsync called with userId={userId}");
-            userCartItems[userId] = emptyCart;
+            userCartItems[userId] = new Hipstershop.Cart();
```

Comment:
It looks like we can remove the emptyCart variable on line 28, right?

Reply:
It's used here -- https://github.com/GoogleCloudPlatform/microservices-demo/blob/e5fb708cf2e5ae9d8a5b71746b1868c6fe9ead8c/src/cartservice/cartstore/LocalCartStore.cs#L80

Reply:
I'm curious whether it would make more sense to just instantiate a new cart there as well. But I'm not really familiar enough with the language to say